templates/layout/siap/menu.html

```diff
@@ -12,7 +12,14 @@
                                         {% for submenu in menu.menuItems %}
                                             <li>
 
-                                                <a class="fr-nav__link" href="{{ request|get_menu_url:submenu.url }}" target="_self" {% if submenu.url|is_conventionnement_menu_url %}aria-current="page"{% endif %}>{{submenu.libelle}}</a>
+                                                <a
+                                                    class="fr-nav__link"
+                                                    href="{{ request|get_menu_url:submenu.url }}"
+                                                    target={% if submenu.openNewWindow %}"_blank"{% else %}"_self"{% endif %}
```

Comment:
Généralement on ajoute le noreferrer si c'est un lien extenre, je ne sais pas si là c'est un autre domaine ? 

Auquel cas on peut ajouter l'attribute 
```suggestion
                                                    {% if submenu.openNewWindow %}target="_blank" rel="noreferrer"{% else %}target="_self"{% endif %}
```

cf https://contentsquare.com/fr-fr/blog/noopener/